chapters/functions.tex

```diff
@@ -2520,6 +2520,11 @@ \subsection{External Objects}\label{external-objects}
   and after the function call).  Normally this is an external function, but it could be a non-external function as well (e.g.\ calling external functions one
   or more times).  The function input shall not have a default value using the constructor.
   \end{nonnormative}
+\item
+  An external object class shall of the specialized type \lstinline!class!.
```

Comment:
Missing _be_ (suggesting minor reformulation), and it's called _specialized class_:
```suggestion
  An external object class shall use the (non-)specialized class \lstinline!class!.
```

Reply:
I prefer "be of", since "use" seems a bit too vague.
Agreed on 'specialized class'.
I also avoid the cute (non)-specialized, since we include 'class' among the specialized classes in other places like https://specification.modelica.org/master/inheritance-modification-and-redeclaration.html#restrictions-on-the-kind-of-base-class
I agree that we likely should add 'class' to the list in https://specification.modelica.org/master/class-predefined-types-and-declarations.html#specialized-classes - but that doesn't seem urgent.

Reply:
OK.  Resolving.